synapse/config/sso.py

```diff
@@ -110,7 +112,10 @@ def generate_config_section(self, **kwargs):
             #client_whitelist:
             #  - https://riot.im/develop
             #  - https://my.custom.client/
-
+            #
+            # Keep a user's display name in sync with the SSO display name
```

Comment:
```suggestion
            # Keep a user's display name in sync with the SSO display name
            #
```

It'd also be good to mention that a user's SSO profile information will only be updated after each log in to Synapse.